docs/ivys.js

```diff
@@ -3,6 +3,14 @@
     browser:true, Promise, firebase, $, jQuery, alert, moment, Spinner, LocalCalendarStorage, FirebaseCalendarStorage
 */
 
+// Helper functions for IvyApp, Product, and ProductCollection classes
+var generateUniqueId = function () {
+    // Unique Ids for tags
+    "use strict";
+    var uniqueId = (Math.floor((Math.random() + Date.now()) * 10e4)).toString();
+    return uniqueId;
+};
+
```

Comment:
Why do tags need unique ids? Don't they have unique names already? What are you calling a `tag` here?